packages/features/portfolio/src/side-drawers/withdraw.tsx

```diff
@@ -42,31 +43,41 @@ export const Withdraw = observer(() => {
   }, [pathname]);
 
   return (
-    <PortfolioSideDrawer isWithdraw>
-      <PortfolioHoldingSelect
-        isWithdraw
-        inputLabel={messages[PORTFOLIO_ACTIONS.WITHDRAW]['inputLabelTwo']}
-      />
-      <DepositInput
-        isWithdraw
-        ref={currencyInputRef}
-        inputRef={currencyInputRef}
-        maxWithdraw={maxWithdrawUnderlying}
-        warningMsg={belowMaxWarning}
-        onMaxValue={onMaxValue}
-        newRoute={(newToken) =>
-          `/portfolio/${selectedNetwork}/${category}/${sideDrawerKey}/${newToken}`
-        }
-        inputLabel={messages[PORTFOLIO_ACTIONS.WITHDRAW]['inputLabel']}
-      />
-      {warning && (
-        <ErrorMessage
-          variant="warning"
-          title={<FormattedMessage {...messages[warning]['title']} />}
-          message={<FormattedMessage {...messages[warning]['message']} />}
-          maxWidth={'100%'}
+    <Container>
+      <PortfolioSideDrawer isWithdraw>
+        <PortfolioHoldingSelect
+          isWithdraw
+          inputLabel={messages[PORTFOLIO_ACTIONS.WITHDRAW]['inputLabelTwo']}
         />
-      )}
-    </PortfolioSideDrawer>
+        <DepositInput
+          isWithdraw
+          ref={currencyInputRef}
+          inputRef={currencyInputRef}
+          maxWithdraw={maxWithdrawUnderlying}
+          warningMsg={belowMaxWarning}
+          onMaxValue={onMaxValue}
+          newRoute={(newToken) =>
+            `/portfolio/${selectedNetwork}/${category}/${sideDrawerKey}/${newToken}`
+          }
+          inputLabel={messages[PORTFOLIO_ACTIONS.WITHDRAW]['inputLabel']}
+        />
+        {warning && (
+          <ErrorMessage
+            variant="warning"
+            title={<FormattedMessage {...messages[warning]['title']} />}
+            message={<FormattedMessage {...messages[warning]['message']} />}
+            maxWidth={'100%'}
+          />
+        )}
+      </PortfolioSideDrawer>
+    </Container>
   );
 });
+
+const Container = styled(Box)(
```

Comment:
Can we put this inside `TransactionSidebar` or `Drawer` instead? I want all the drawers to appear the same way on mobile.